app/test_main.py

```diff
@@ -1 +1,45 @@
-# write your code here
+from unittest.mock import patch
+from typing import Any
+
+from pytest import mark
+from pytest import fixture
+from pytest import param
+
+from app.main import can_access_google_page
+
+
+@fixture()
+def income_correct_url() -> str:
+    return "https://www.google.com/"
+
+
+@fixture()
+def income_incorrect_url() -> str:
+    return "https://www.chatgpt.com/"
```

Comment:
Fixtures are better to use if the URLs need to be fetched from external sources or some dynamically generated.
Here will be better to use some constant f.e.:
```
CORRECT_URL = "https://www.google.com/"
INCORRECT_URL = "https://www.chatgpt.com/"
```